scripts/fast-reboot

```diff
@@ -462,21 +462,14 @@ function unload_kernel()
 }
 
 function save_counters_folder() {
-    debug "Saving counters folder before warmboot..."
+    if [[ "$REBOOT_TYPE" = "warm-reboot" ]]; then
```

Comment:
Why don't do the backup for `fastfast-reboot`?

Reply:
```fastfast-reboot``` does not save counter values, at least for some of them. We prefer not to make a backup for ```fastfast-reboot```. In this case, counters behave like in ```fast-reboot```